BabbleApp/requirements.txt

```
@@ -1,6 +1,6 @@
-onnxruntime==1.19.2; 
+onnxruntime==1.20.1; 
```

```suggestion
onnxruntime==1.19.2; 
```

Currently, `onnxruntime` wheels do not exist for Python 3.13. As such, this needs to remain on 1.19.2. See: https://github.com/Project-Babble/ProjectBabble/issues/87